### PR TITLE
HPCC-30450 Add dependabot.yml and bump versions

### PR DESCRIPTION
**Pull request by GordonSmith**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by ghalliday** [comment]
Will let Jeremy comment on the ts changes - are they because a version was bumped.
Is this targeting the correct version?  I imagine we need some changes targeting 9.4.x and others earlier.

File: esp/src/src-react/theme-shims/v9ThemeShim.ts [hunk 1]
```diff
@@ -34,7 +33,7 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralForeground3BrandPressed: palette.themeDarkAlt,
     colorNeutralForeground3BrandSelected: palette.themePrimary,
     colorNeutralForeground4: palette.neutralQuaternary,
-    colorNeutralForegroundDisabled: palette.neutralQuaternaryAlt,
+    colorNeutralForegroundDisabled: palette.neutralTertiaryAlt,
```
Comment (jeclrsg) [inline esp/src/src-react/theme-shims/v9ThemeShim.ts:36]: The changes to these colors on line 36 & 112 would undo some changes made in #17743

**Review by jeclrsg — CHANGES_REQUESTED** [review]
@GordonSmith Looked good. Just noted that there's a change in the v9ThemeShim which would undo changes from a few weeks back (I believe it was to make disabled pivots be a little more obvious they weren't clickable)

**Comment by GordonSmith** [comment]
Closing as split out dependabot.yml into single PR

**Pull request closed without merging**